gradle-plugin/src/test/groovy/io/realm/gradle/PluginTest.groovy

```diff
@@ -108,6 +108,71 @@ class PluginTest {
         }
     }
 
+    @Test
+    void pluginAddsRightRepositories_noRepositorySet() {
+        project.buildscript {
+            dependencies {
+                classpath "com.android.tools.build:gradle:${projectDependencies.get("GRADLE_BUILD_TOOLS")}"
+                classpath 'com.jakewharton.sdkmanager:gradle-plugin:0.12.0'
+            }
+        }
+
+        def manifest = project.file("src/main/AndroidManifest.xml")
+        manifest.parentFile.mkdirs()
+        manifest.text = '<manifest xmlns:android="http://schemas.android.com/apk/res/android"  package="com.realm.test"></manifest>'
+
+        project.apply plugin: 'com.android.application'
+        project.apply plugin: 'realm-android'
+
+        project.android {
+            compileSdkVersion 27
+
+            defaultConfig {
+                minSdkVersion 16
+                targetSdkVersion 27
+            }
+        }
+
+        project.evaluate()
+
+        assertTrue(project.repositories.size() == 1)
+        assertTrue(project.repositories.contains(project.getRepositories().jcenter()))
```

Comment:
❌ This doesn't test anything. The fact of calling `project.repositories.jcenter()` returns a `jcenter` `ArtifactRepository` instance but also adds it to the container. This assertion is a no-op in this case. A better way to assert is looking that the host of the URL of the first repository is `jcenter.bintray.com`. Something like this would work: `assertTrue(project.repositories.first().url.host == 'jcenter.bintray.com')`.

Reply:
Cool. Thanks for clarifying